CONRTIBUTING.md

```diff
@@ -61,6 +61,28 @@ __The following approach aims to reconcile these two competing demands:__
 --------
 
 
+### Library Standards
+
+The team has agreed to consistently use several third-party Python packages.
+This list will likely be fairly dynamic as we are always looking for things
+that will increase our productivity. We have a strong preference for tools that
+follow the [Principle of Least Astonishment](http://c2.com/cgi/wiki?PrincipleOfLeastAstonishment)
+(also on [Wikipedia](https://en.wikipedia.org/wiki/Principle_of_least_astonishment)).
+
+The list:
+
+  - [arrow](https://pypi.python.org/pypi/arrow): Date/Time utilities
```

Comment:
I don't think there are limitations as much as there is questionable ease-of-use.